examples/axis_inputs.rs

```diff
@@ -33,9 +33,10 @@ fn spawn_player(mut commands: Commands) {
         .insert(Action::Throttle, GamepadButtonType::RightTrigger2)
         // And we'll use the right stick's x-axis as a rudder control
         .insert(
+            // Add an AxisDeadzone to process horizontal values of the right stick.
             // This will trigger if the axis is moved 10% or more in either direction.
             Action::Rudder,
-            SingleAxis::symmetric(GamepadAxisType::RightStickX, 0.1),
+            SingleAxis::new(GamepadAxisType::RightStickX).with_processor(AxisDeadzone::default()),

```

Comment:
This will be clearer and more robust if we specify the actual value, rather than relying on the `Default` value aligning.